### DISCUSSION TITLE
亮度调节无效

### DISCUSSION THREAD

**Discussion by leimuhhh** [shuax/MouseInc.Settings#93] [Q&A]
你好，2.11.4还能调节亮度，今天更新完2.11.5亮度调节无效了，系统亮度可以调节

**Comment by shuax** [comment]
什么系统，装杀毒软件了吗

**Reply by leimuhhh** [reply]
没有杀软，刚才更新了2.11.6，调节亮度的那个速度是回复正常了，都是亮度还是没有变化

**Reply by leimuhhh** [reply]
win10 1903

**Reply by rolandfang** [reply]
更新了2.11.6b以后我的扩展显示器无法调节屏幕亮度，滚动鼠标滚轮可以触发动作，但是无效果，而笔记本的屏幕可以正常调节亮度。系统版本win10,2004,没有安装第三方杀毒软件，用的是widonws安全中心。

**Comment by shuax** [comment]
新版的目前只能调节主屏幕。

**Reply by leimuhhh** [reply]
目前还是不能调节亮度

**Reply by rolandfang** [reply]
建议后续版本增加对扩展屏亮度调节的功能。

**Reply by leimuhhh** [reply]
希望大佬想想办法，每次进设置调好麻烦
